### PR TITLE
Upgrade to gridstack 2.0

### LINKED ISSUE #48: Update to gridstack 2.0.0
`gridstack@v2.0.0` was just released: https://github.com/gridstack/gridstack.js/releases/tag/v2.0.0

We should look into the places where it is referenced and start updating to the new version.

- `v1.0.0` vendored here: 

https://github.com/voila-dashboards/voila-gridstack/blob/0c9ce7909b61bd86478c10b3ed25b590ae2b35a1/voila-gridstack/static/gridstack.js#L2

- `v0.5.2` referenced here:

https://github.com/voila-dashboards/voila-gridstack/blob/0c9ce7909b61bd86478c10b3ed25b590ae2b35a1/share/jupyter/nbconvert/templates/gridstack/gridstack_base.html.j2#L4

- JQuery UI Drag&Drop plugin `v1.0.0-dev` referenced here:

https://github.com/voila-dashboards/voila-gridstack/blob/0c9ce7909b61bd86478c10b3ed25b590ae2b35a1/voila-gridstack/static/gridstack.jqueryUI_require.js#L1

We could start using the same version everywhere to keep the same API across the code base.

### PR DESCRIPTION
**Pull request by jtpio**
Fixes #48. 

### TODO

- [x] Update template
- [x] Update classic notebook extension
- [x] Fix warnings (deprecated attributes)
- [x] Update the tests

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -52,7 +52,6 @@ def get_data_files():
                                                         'voila-gridstack/static/voila-gridstack.js',
                                                         'voila-gridstack/static/voila-gridstack.css',
                                                         'voila-gridstack/static/gridstack.js',
-                                                        'voila-gridstack/static/gridstack.jqueryUI_require.js'
```
Comment (SylvainCorlay) [inline setup.py:55]: :tada:

**Comment by jtpio** [comment]
We can tackle https://github.com/voila-dashboards/voila-gridstack/issues/26 in a separate PR.

File: setup.py (hunk 1 above)
Comment (SylvainCorlay) [reply setup.py:55]: @jtpio should we keep the listing of gridstack.js above?

Comment (jtpio) [reply setup.py:55]: Looks like it's still needed for the classic notebook extension:

https://github.com/voila-dashboards/voila-gridstack/blob/29e9569263cbfc8ac7aef7071cd73cbb09dcc2d3/voila-gridstack/static/extension.js#L13

Comment (SylvainCorlay) [reply setup.py:55]: But you delete the gridstack.js file from the repository?

Comment (jtpio) [reply setup.py:55]: It's not deleted by updated with the new (minified) version.

**Comment by jtpio** [comment]
We would need to do a pass on the CSS for the classic notebook extension as it seems to be a bit off now:

image

File: voila-gridstack/static/extension.js [hunk 2]
```diff
@@ -11,13 +11,18 @@
...
-       function($, Jupyter, gridstack, _, voila_gridstack) {
+       function($, Jupyter, gridstack, voila_gridstack) {
+
+    var GRIDSTACK_STYLES = 'https://cdn.jsdelivr.net/npm/gridstack@2.0.0/dist/gridstack.min.css';
+    var grid;
 
     function load_ipython_extension() {
 
+        // import gridstack styles
+        $('head').append( $('<link rel="stylesheet" type="text/css" />').attr('href', GRIDSTACK_STYLES) );
```
Comment (jtpio) [inline voila-gridstack/static/extension.js:24]: Loading the stylesheet once on init should fix the issue mentioned in https://github.com/voila-dashboards/voila-gridstack/pull/56#issuecomment-690315751

For reference, the GridStack CSS is scoped to `.grid-stack`: 

https://github.com/gridstack/gridstack.js/blob/cf393169d3961abd35d6570c3e08b429a7341cfb/src/gridstack.scss#L21

Comment (SylvainCorlay) [reply voila-gridstack/static/extension.js:24]: awesome!

Comment (SylvainCorlay) [reply voila-gridstack/static/extension.js:24]: I marked your earlier comment as resolved.

File: setup.py (hunk 1 above)
Comment (SylvainCorlay) [reply setup.py:55]: Sorry, I am confused. I see the file deleted in the PR.

**Pull request merged**